boote/templates/base_boote.html

```diff
@@ -43,7 +43,7 @@
     <ul class="dropdown-menu">        
         <li><a class="dropdown-item" href="/boote/booking/today/">Verfügbarkeit Heute</a></li>        
         <li><a class="dropdown-item" href="/boote/booking/overview/">Termin Reservieren</a></li>        
-        <li><a class="dropdown-item" href="/boote/booking/my_bookings/">Meine Reservierungen</a></li>
+        <li><a class="dropdown-item" href="/boote/booking/my_bookings/">Termin Stornieren</a></li>
```

Comment:
"stornieren" should be written with small "s", we could also update "heute" and "reservieren"